<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumeralSystem</class>
 <widget class="QMainWindow" name="NumeralSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>287</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>287</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/logo.png</normaloff>img/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>261</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>261</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"> background: #202124</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Numbers">
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_0">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_nine">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_eight">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_1">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_six">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_reset">
       <property name="styleSheet">
        <string notr="true">background: #5f6368;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pushButton_equ">
       <property name="styleSheet">
        <string notr="true">background: #5f6368;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_seven">
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">    background: #3c4043;
    color: #e8eaed;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="INPUT">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="INFO_INP">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>INPUT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="input">
       <property name="styleSheet">
        <string notr="true">    background: #FFFFFF;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>20</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="OUTPUT">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="INFO_OUT">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>OUTPUT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="output">
       <property name="styleSheet">
        <string notr="true">    background: #FFFFFF;
    border: 1px solid #5f6368;
    border-radius: 4px;
    box-sizing: border-box;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="FROM">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>FROM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QRadioButton" name="pushButton_TN16">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>N₁₆</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="pushButton_FN2">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>N₂</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QRadioButton" name="pushButton_TN2">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>N₂</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="pushButton_FN10">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>N₁₀</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QRadioButton" name="pushButton_TN10">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>N₁₀</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="pushButton_FN8">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>N₈</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3" rowspan="4">
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="TO">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>TO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="4">
      <widget class="Line" name="line_3">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="3">
      <widget class="QLabel" name="picture">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="pixmap">
        <pixmap resource="reusrces.qrc">:/res/img/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="CONVERT">
       <property name="styleSheet">
        <string notr="true">    color: #e8eaed;</string>
       </property>
       <property name="text">
        <string>CONVERT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Fonk">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>-1</y>
      <width>631</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background: #202124
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>Fonk</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuoperating_mode_2">
     <property name="title">
      <string>operating mode</string>
     </property>
     <addaction name="defoultcalculator_swap"/>
     <addaction name="actionnumeral_system_current"/>
    </widget>
    <widget class="QMenu" name="menuDon_t_open">
     <property name="title">
      <string>Don't open</string>
     </property>
     <addaction name="actionNew_Pic"/>
    </widget>
    <addaction name="menuoperating_mode_2"/>
    <addaction name="menuDon_t_open"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actiondefoult_calculator">
   <property name="text">
    <string>defoult calculator</string>
   </property>
  </action>
  <action name="actionnumeral_system">
   <property name="text">
    <string>numeral system</string>
   </property>
  </action>
  <action name="actionnumeral_system_current">
   <property name="text">
    <string>numeral system (current)</string>
   </property>
  </action>
  <action name="defoultcalculator_swap">
   <property name="text">
    <string>defoult calculator</string>
   </property>
  </action>
  <action name="actionNew_Pic">
   <property name="text">
    <string>DON`T OPEN THIS</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="reusrces.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
